.prettierrc

```diff
@@ -4,5 +4,6 @@
   "tabWidth": 2,
   "printWidth": 120,
   "semi": true,
-  "arrowParens": "avoid"
+  "arrowParens": "avoid",
+  "endOfLine": "auto"
```

Comment:
```suggestion
  "endOfLine": "lf"
```
The "auto" can introduce a confusion - it would allow for both \n and \r\n - files created on Windows will have the latter format, and created on Mac/Linux the former.

Reply:
I changed CRLF to LF, but doesn't git automatically change all LF's back to CRLF? 
![image](https://user-images.githubusercontent.com/56044700/127753831-dc274e15-2288-4732-95e4-34f4d1afcb40.png)

Reply:
Ok, leave the "auto" for now, I'll try it out on non-windows systems later